Program.cs

```diff
@@ -319,7 +309,6 @@ private static void ReportCustomersAge()
             MenuMain();
             ProcessMain();
         }
-
         private static void ReportCustomersBirthay()
         {
             List<Customers> objCustomersList = new List<Customers>();

```

Comment:
You are not using the new instance created.

Reply:
corrected